bigbluebutton-html5/imports/startup/server/userPermissions.js

```diff
@@ -111,6 +112,9 @@ const viewer = function (meetingId, userId) {
     setEmojiStatus: true,
     clearEmojiStatus: true,
 
+    // user control
+    setPresenter: true,
```

Comment:
We don't actually want any viewer to be able to set who the presenter is.
Only moderators should be able to see the setPresenter row in the userlist dropdown.